lib/ansible/modules/cloud/amazon/cloudfront_distribution.py

```diff
@@ -1864,6 +1875,10 @@ def main():
             ['default_origin_domain_name', 'alias'],
         ]
     )
+    if not module.botocore_at_least('1.9.22'):
```

Comment:
I do address this in the PR description. It's not possible to delete a distribution without updating it first to disable it. It would be possible to create one, but the logic of creation/update is so intertwined that it's easier to just enforce this.